project-base/storefront/components/Layout/OrderLayout/OrderLayout.tsx

```diff
@@ -25,17 +27,23 @@ export const OrderLayout: FC<OrderLayoutProps> = ({ activeStep, isTransportOrPay
                 <Header simpleHeader />
             </Webline>
             <Adverts positionName="header" withGapBottom withWebline />
-            <Webline>
-                <OrderSteps activeStep={activeStep} domainUrl={url} />
-            </Webline>
-            <Webline>
-                <div className="mb-24 flex w-full flex-col flex-wrap vl:mt-7 vl:mb-16 vl:flex-row">
-                    <div className="mb-16 w-full vl:mb-0 vl:min-h-[61vh] vl:flex-1 vl:pr-10">{children}</div>
-                    <div className="w-full vl:max-w-md">
-                        <OrderSummary isTransportOrPaymentLoading={isTransportOrPaymentLoading} />
-                    </div>
-                </div>
-            </Webline>
+            {isLoading ? (
+                <OrderLayoutContentSkeleton>{children}</OrderLayoutContentSkeleton>
+            ) : (
+                <>
+                    <Webline>
+                        <OrderSteps activeStep={activeStep} domainUrl={url} />
+                    </Webline>
+                    <Webline>
+                        <div className="mb-24 flex w-full flex-col flex-wrap vl:mt-7 vl:mb-16 vl:flex-row">
+                            <div className="mb-16 w-full vl:mb-0 vl:min-h-[61vh] vl:flex-1 vl:pr-10">{children}</div>
+                            <div className="w-full vl:max-w-md">
+                                <OrderSummary isTransportOrPaymentLoading={isTransportOrPaymentLoading} />
+                            </div>
+                        </div>
+                    </Webline>
+                </>
+            )}
```

Comment:
Maybe you can remove Webline from skeleton and from content shown after loading is finished and simply wrap to Webline all of this?

```suggestion
            <Webline>
            {isLoading ? (
                <OrderLayoutContentSkeleton>{children}</OrderLayoutContentSkeleton>
            ) : (
                <>
                        <OrderSteps activeStep={activeStep} domainUrl={url} />
                        
                        <div className="mb-24 flex w-full flex-col flex-wrap vl:mt-7 vl:mb-16 vl:flex-row">
                            <div className="mb-16 w-full vl:mb-0 vl:min-h-[61vh] vl:flex-1 vl:pr-10">{children}</div>
                            <div className="w-full vl:max-w-md">
                                <OrderSummary isTransportOrPaymentLoading={isTransportOrPaymentLoading} />
                            </div>
                        </div>
                </>
            )}
            </Webline>
```